docs/slack.rst

```diff
@@ -75,6 +80,7 @@ please avoid these behaviors:
 - **Let Me Google That For You (LMGTFY)** - If people ask questions that are simple to answer on Google, still answer them nicely.
 - **Question Policing** - Some people might ask incomplete questions that are hard to answer. Ask follow up questions and act in good faith to help them solve their issues.
 - **Technology Competitions** - We don't care if your tool is faster or better in some metric. We care about creating great documentation for our users.
+- **Spam** - Respect our space. Don't post spam or otherwise vandalize our community.
```

Comment:
I don't think this item addresses the post/s that troubled you recently, Mike. IOW, I doubt that the poster would recognize their post as spam. There's a slippery slope with contributors falling into promotional posts, but up to now it's been OK bc said posters have also been real participants in the community. The recent incident wasn't like this, and I think we want to discourage such things, but I'm not sure this will help.

Reply:
How would you address such promotional posts? 

The original post that I was concerned about included the following phrase: "we’re constantly posting across a variety of platforms " I think that's a pretty explicit acknowledgement of spam.